demisto_sdk/tests/test_files/playbook-Invalid_condition_1.yml

```diff
@@ -0,0 +1,111 @@
+elasticcommonfields: {}
+id: Invalid_condition_1
```

Comment:
for both playbooks with invalid conditions, lets write why they're invalid

Reply:
changed to `Invalid_condition_unhandled_branch` and `Invalid_condition_unhandled_branch_and_unhandled_condition`